### DISCUSSION TITLE
Visually separate 'Post' button from other buttons

### DISCUSSION THREAD

**Discussion by ehuelsmann** [ledgersmb/LedgerSMB#5858] [Ideas]
The feature idea here is to separate the 'Post' button visually from the others by

* Putting it at a visual distance
* Giving it a different color (e.g. 'green' where the rest of the buttons are blue)

This way, the "Post" button isn't "just another button", but stands out which helps to indicate its "no way back" behaviour.

**Comment by einhverfr** [comment]
agreed.

On Sat, Dec 10, 2016 at 7:27 PM, Erik Huelsmann <notifications@github.com>
wrote:

> The feature idea here is to separate the 'Post' button visually from the
> others by
>
>    - Putting it at a visual distance
>    - Giving it a different color (e.g. 'green' where the rest of the
>    buttons are blue)
>
> This way, the "Post" button isn't "just another button", but stands out
> which helps to indicate its "no way back" behaviour.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ledgersmb/LedgerSMB/issues/2368>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAU0lwE5qomVJRhHoApZFD_4Me-rokSoks5rGu8NgaJpZM4LJuRf>
> .
>



-- 
Best Wishes,
Chris Travers

Efficito:  Hosted Accounting and ERP.  Robust and Flexible.  No vendor
lock-in.
http://www.efficito.com/learn_more
